### PR TITLE
Add the Ortho60 v2 Keyboard to QMK

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Adding the Ortho60v2 keyboard to QMK.
I made it a new top-level keyboard because of the addition of an encoder.
MCU has also been swapped to RP2040

A revision of the existing top level Ortho60 didn't feel quite appropriate due to these changes - but if necessary, I can change organization to reflect this.

Targeting develop so we can use the PWM functionality @KarlK90 has added :)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/ortho60v2/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _RAISE 1
+#define _LOWER 2
```
Comment (drashna) [inline keyboards/cannonkeys/ortho60v2/keymaps/default/keymap.c:26]:
```suggestion
enum custom_layers {
    _BASE,
    _RAISE,
    _LOWER,
};
```

File: keyboards/cannonkeys/ortho60v2/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (drashna) [inline keyboards/cannonkeys/ortho60v2/keymaps/via/rules.mk:2]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by awkannan** [comment]
@drashna Ready for re-review :) Thank you!

File: keyboards/cannonkeys/ortho60v2/ortho60v2.c [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#include "ortho60v2.h"
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+        if (clockwise) {
+            tap_code_delay(KC_VOLU, 10);
+        } else {
+            tap_code_delay(KC_VOLD, 10);
+        }
+    return encoder_update_user(index, clockwise);
+}
```
Comment (drashna) [inline keyboards/cannonkeys/ortho60v2/ortho60v2.c:26]:
```suggestion
#ifdef ENCODER_ENABLE
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
        return false;
    }
    if (clockwise) {
        tap_code_delay(KC_VOLU, 10);
    } else {
        tap_code_delay(KC_VOLD, 10);
    }
    return true;
}
#endif
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/cannonkeys/ortho60v2/readme.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Ortho60 v2
+
+A 5x12 Ortholinear keyboard powered by an onboard RP2040
+
+* Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan1)  
+* Hardware Supported: RP2040 
+* Hardware Availability: [CannonKeys](https://cannonkeys.com)
```
Comment (fauxpark) [inline keyboards/cannonkeys/ortho60v2/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan1)
* Hardware Supported: RP2040
* Hardware Availability: [CannonKeys](https://cannonkeys.com)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**